dev_guide/deployments.adoc

```diff
@@ -352,3 +352,16 @@ Rollbacks revert an application back to a previous deployment and can be
 performed using the REST API or the CLI. See the
 link:../cli_reference/basic_cli_operations.html#deployment-operations[CLI
 Reference] for more details.
+
+== Fine-Grained Control
```

Comment:
@adellape
Good question.  Here are my thoughts:

There are two phases of interaction w/ the "deployment config" object: creation and maintenance (runtime tweaking).

My assumption about creation phase was incorrect (see upthread) -- gist is: ideally, there is no v0 dep.yaml created by a human and loaded into openshift; instead, some (hand wave) creates this object internally and if concrete on-disk representation is desired, it must be extracted afterwards (perhaps in maintenance phase).

Maintenance is done in two ways: whole-cloth and fine-grained.
Whole-cloth is basically covered by "osc update -f vN+1.yaml".
Fine-grained is "osc resize", "osc label" (i think), and so on (more ignorant hand-waving here).

So, choosing "fine-grained" reflects this kind of forward thinking.
I don't know if it's the "right" kind of forward thinking, though.  :-D